platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,515 @@
+
+##### Characteristics:
+
+* The team is made up of nearly all technical generalists
+* Team budget may include the infrastructure costs associated with their work leading to often being a key point in budget conversations
+* Backlog items range a number of technologies, leading to frequent and large context switches. 
+* This team is often the first to fill a gap that is not yet being addressed, even if not in the declared scope for the team - this team takes ownership of resources that don't have an owner
+* Assigned people rarely have the time or experience with customer research to validate their designs or implementations
+
+##### Examples Scenarios:
+
+* Application developers raise an issue with the long build time for their applications. A centralized team is tasked with reducing the build time by 50%. They solve this by doubling the size and quantity of the CI runners given they are not close enough to the software to individually improve the application builds. This creates a budget concern for their centralized team as the productivity gain is not directly measurable against this increased infrastructure cost.
+
+#### Level 3, Scalable - As product
+
+Investment in internal platforms and their capabilities is similar to investment in an enterprise's outbound products and value streams: based on the value they are expected to provide to their customers. Product management and user experience are explicitly considered and invested in. A chargeback system may be used to reflect platforms' impact on their customers' own direct value streams and products. The enterprise allocates funds and staff to the appropriate initiatives by using data-driven performance indicators and feedback loops. Platform teams can ultimately optimize the business itself and contribute to increased profitability.
+
+##### Characteristics:
+
+* Platform teams staff roles not traditionally found in internal serving or technical teams, for example, product management and user experience
+* The team publicizes a roadmap internally to the organization which indicates the value delivered and high level feature targets
+* Features are tested for both implementation quality and user experience during design, delivery, and post deployment
+* Feature removal is a key part of the conversation, the goal is to have a well supported, well used suite of capabilities instead of a sprawling estate that may not be maintained
```

Comment:
```suggestion
* Feature removal is a key part of the conversation, the goal is to have a well supported, well used suite of capabilities instead of a sprawling estate that may not be maintained.
```